scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -339,7 +339,7 @@ <h5 class="modal-title" id="case_pick_dialog">Add other case to case group ({{ l
 <!-- Iterate through Individuals, if chromograph_files is presenet add a panel for images -->
 {% set chromograph = namespace(available=false) %}
 {% macro insert_multi_image_panel() %}
-<div class="panel-group" id="accordion">
+<div class="panel-group no-animation" id="accordion">
```

Comment:
Where does this no-animation class come from? Is it a custom style or a bootstrap one? If it's custom I don't find it anywhere

Reply:
Reviewer 2 is always right! 😊  I was sure it was in the css above, but probably two different versions that diverged a little. The class should ideally be added to the css instead of disabling transition for all collapses on the page.

Reply:
Reviewer 2 is such an evil person, when it comes to manuscript revisions at least! 😆

Reply:
> Reviewer 2 is always right! 😊 I was sure it was in the css above, but probably two different versions that diverged a little. The class should ideally be added to the css instead of disabling transition for all collapses on the page.

Yes. That was the plan och also the implementation. Might differ from committed code.

Reply:
Now committed code and intention are the same.